### ISSUE TITLE
Missing form label error on validating ComboBox, DropDownList, MultiSelect for Section 508 compliance

### ISSUE THREAD

**Issue by Dimitar-Goshev** [telerik/kendo-ui-core#2757]
### Bug report
Although the recommendations from the [Section 508 - Label Element Support](http://docs.telerik.com/kendo-ui/accessibility/section-508#label-element-support) help article are applied and the respective widgets would be focused on clicking their labels, still there are Section 508 compliance errors - Missing form label error

### Reproduction of the problem
Open the following [Dojo](http://dojo.telerik.com/aqeLA/3) in [Full screen mode](http://runner.telerik.io/fullscreen/aqeLA/3) in Chrome and validate with Wave extension OR load the [Full screen mode](http://runner.telerik.io/fullscreen/aqeLA/3) URL in [WAVE Validator](http://wave.webaim.org/) - [result](http://wave.webaim.org/report#/http://runner.telerik.io/fullscreen/aqeLA/3)

### Current behavior
Missing form label errors - [screenshot](https://www.screencast.com/t/ry18XkcC)

### Expected/desired behavior
No error - [screenshot](https://www.screencast.com/t/WRDAs1Em), as in the case of NumericTextBox, which is shown in the help article.

### Environment

* **Kendo UI version:** 2017.1.118
* **Browser:** [all]

**Comment by veselints** [comment]
The issue is caused by the fact, that the validator looks for properly accossiated <label> element for each <input> or <select> using the <lable for=""> attribute. The validation passes when using this attribute. This, houwere, does not focus the ComboBox and the MultiSelect on clicking on the label.

Also, the MultiSelect remains invalid. This issue is caused by the fact, that within the widget there are an <input> element and a <select> element. The <select> is hidden and holds the id of the widget, so the input element remains without label. Possible workarownd for this issue could be as implemented in [the following Dojo](http://dojo.telerik.com/UqeFU/24). It simply injects a second <lable> element for the <input> field.

**Comment by veselints** [comment]
Increasing severity. Reported in Ticket 1098145

**Comment by veselints** [comment]
Fixed with [commit](https://github.com/telerik/kendo-ui-core/pull/3143/commits/0df2851a41b52e51c056d0abcc1284cfbc2ab462)
